mempool/clist_mempool.go

```diff
@@ -674,3 +650,20 @@ func (mem *CListMempool) recheckTxs() {
 	// all pending messages to the app. There doesn't seem to be any need here as the buffer
 	// will get flushed regularly or when filled.
 }
+
+func (mem *CListMempool) noRecheckingRequests() bool {
+	return atomic.LoadUint32(&mem.numRecheckingRequests) == 0
+}
+
+func (mem *CListMempool) increaseNumRecheckingRequests() uint32 {
+	mem.logger.Info("increaseNumRecheckingRequests")
```

Comment:
Can we make this log more readable?

Reply:
This log was not supposed to be here. Removed.